CODEOWNERS

```diff
@@ -1,18 +1,15 @@
 # Generally we are following a permissive policy of code ownership
-# Some people have asked for only minimal permissions to avoid notification spam: @bgrant0607
-
-# Default, unless something takes precedence below
-* @droot @justinsb @mortent @natasha41575 @yuwenma
+# We use the @GoogleContainerTools/kpt-reviewers group so that we can assign a single person to each PR.
 
 # Docs
-/site @bgrant0607 @droot @justinsb @mortent @natasha41575 @yuwenma
+/site @GoogleContainerTools/kpt-reviewers
```

Comment:
Do we need to specify separate owners for all these paths, when we use the same group for all of them?